projects/vdk-core/src/vdk/internal/builtin_plugins/config/job_config.py

```diff
@@ -44,23 +44,25 @@ class JobConfig:
 
     def __init__(self, data_job_path: Union[pathlib.Path, str]):
         self._config_ini = configparser.ConfigParser()
-        self._config_file = os.path.join(data_job_path, "config.ini")
-
-        if not os.path.isfile(self._config_file):
-            raise VdkConfigurationError(
-                ErrorMessage(
-                    summary="Error while loading config.ini file",
-                    what="Cannot extract job Configuration",
-                    why=f"Configuration file config.ini is missing in data job path: {data_job_path}",
-                    consequences="Cannot deploy and configure the data job without config.ini file.",
-                    countermeasures="config.ini must be in the root of the data job folder. "
-                    "Make sure the file is created "
-                    "or double check the data job path is passed correctly.",
-                )
-            )
-        self._read_config_ini_file(
-            config_parser=self._config_ini, configuration_file_path=self._config_file
-        )
+        self._config_ini.read(str(data_job_path.joinpath("config.ini")))
+
+        # TODO: Figure out a way to properly handle cases where there is no config.ini present
+        # self._config_file = os.path.join(data_job_path, "config.ini")
+        # if not os.path.isfile(self._config_file):
+        #     raise VdkConfigurationError(
+        #         ErrorMessage(
+        #             summary="Error while loading config.ini file",
+        #             what="Cannot extract job Configuration",
+        #             why=f"Configuration file config.ini is missing in data job path: {data_job_path}",
+        #             consequences="Cannot deploy and configure the data job without config.ini file.",
+        #             countermeasures="config.ini must be in the root of the data job folder. "
+        #             "Make sure the file is created "
+        #             "or double check the data job path is passed correctly.",
+        #         )
+        #     )
+        # self._read_config_ini_file(
+        #     config_parser=self._config_ini, configuration_file_path=self._config_file
+        # )
```

Comment:
I feel you have commented out more than needed. Shouldn't only the raise VdkConfigurationError be commented. 

Also you may add log.info statement.

Reply:
You're correct. I refactored it to only log a message and not throw an error. Thank you for the review